app/views/admin/categories/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Categories</h1>
+<%= form_for Category.new ,url:admin_categories_path(@admin, @category) do |f| %>
+<div class="form-group">
+  <%= f.label:name %>
+  <%= f.text_field :name, class:'form-control',placeholder:"enter your name" %>
+</div>
+<%= f.submit 'Submit', class: 'btn btn-primary' %>
+<% end %>
+<%= link_to 'Home',root_path %>
```

Comment:
1. Lỗi indentation nha em
2. `@admin` em lấy đâu ra thế, và có cần không?
3. `url:admin_categories_path` cái này có cần truyền gì vào không? Nếu bỏ hết tham số đi thì có lỗi không?

Reply:
2.@admin em đọc trong tài liệu nói : có thể sử dụng @admin để thay thế các số IDs. link:https://guides.rubyonrails.org/routing.html#creating-paths-and-urls-from-objects
- và cũng ko cần thiết trong trường hợp này . bỏ đi vẫn chạy bình thường
3. cái này ko cần truyền gì vào , nếu bỏ hết tham số đi thì không chạy được